src/layouts/SearchView.js

```diff
@@ -1,38 +1,41 @@
-import React, { Component } from 'react';
-import 'react-virtualized/styles.css'; // only needs to be imported once
+import React, { Component } from "react";
+import "react-virtualized/styles.css"; // only needs to be imported once
 import { connect } from "react-redux";
-import { PhotoListView } from '../components/photolist/PhotoListView'
+import { PhotoListView } from "../components/photolist/PhotoListView";
+import { Photoset } from "../reducers/photosReducer";
+
 export class SearchView extends Component {
-    render() {
-        console.log(this.props.searchPhotosResGroupedByDate)
-        const {searchingPhotos,searchQuery} = this.props
-        return (
-            <PhotoListView 
-                title={searchingPhotos ? `Searching "${searchQuery}"...` : searchQuery===null ? "Search for things, places, people, and time." : `"${searchQuery}"`}
-                loading={searchingPhotos}
-                titleIconName={'search'}
-                isDateView={true}
-                photosGroupedByDate={this.props.searchPhotosResGroupedByDate[0] ? this.props.searchPhotosResGroupedByDate : []}
-                idx2hash={this.props.searchPhotosResGroupedByDate[0] ? this.props.searchPhotosResGroupedByDate.flatMap((el) => el.items) : []}
-            />
-        )
-    }
+  render() {
+    // I don't know when the alternative titles would be shown, since the
```

Comment:
I think this was the title when you would navigate to /search without a query parameter, and I probably regressed on that while I was refactoring PhotoListView.

I think it would make more sense, if search without a query parameter would redirect you.

Reply:
Right. I agree. A redirect to '/', I assume?

Reply:
Code updated to redirect to / when the user navigates to /search without a query.